integration_tests/meta_eval_nlg_test.py

```diff
@@ -189,3 +182,74 @@ def test_dataset_level_spearmanr_bootstrap(self) -> None:
         ci = unwrap(corr_metric.calc_confidence_interval(stats, 0.05))
         self.assertAlmostEqual(ci[0], 1, 2)
         self.assertAlmostEqual(ci[1], 1, 2)
+
+
+class MetaEvalNLGNewsroomTest(unittest.TestCase):
+    """
+    Test the NLG metric on newsroom dataset and replicate the reported results from
+    the paper: https://arxiv.org/pdf/2106.11520.pdf
```

Comment:
Could you please add some of the details from the PR into this comment here so they remain for later?

Also, I agree with Yusuke's comments below that it'd be a good idea to specify which results were replicated.

Reply:
>"To truly replicate the results we can probably calculate ROUGE with the same stemmer as the original paper. Would that be hard?"

I won't plan to do this.  (1) We need to modify the eaas code. (2) the original paper use a package that will be outdated. no need to support it.  Anyway, no plan to do this here.